lib/api/client/query.ts

```diff
@@ -18,12 +20,21 @@ if (
 const queryAPI = async <Query, Variables extends AnyVariables = AnyVariables>({
   query,
   variables,
+  fetchOptions: inputFetchOptions,
```

Comment:
expose fetch options to through our API client, mainly for tagging fetches for revalidation